test/logs_aggregator/start_log_aggregator.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+echo "working in ${DIR}"
+cd $DIR && pwd
+logs="$(ls -1 '../../tmp_log/')"
+path="$(readlink -f '../../tmp_log/')"
+echo "Current localnet logs are placed into '${path}/${logs}'"
+echo "CURRENT_SESSION_LOGS='${path}/${logs}'" > .env
+echo "CURRENT_SESSION_LOGS='${path}/${logs}'"
+echo "starting docker compose lor log aggregation"
+docker-compose up --detach
```

Comment:
i believe the latest version of docker docker-compose is no longer using `docker-compose` but `docker compose`

Reply:
Hey, `docker-compose` backward compatible, but I agreed to use the new one, less confusion + it will supported in the future releases

Reply:
Backward compatible, but agreed, new release can remove deprecated command